es/index.md

```diff
@@ -1,110 +1,91 @@
 ## Presentaciones listas para usar
 
-Esta es una colección de presentaciones que puedes usar, adaptar, y modificar para *tu* propia presentacion! Esta construida por Cloud Advocates de Microsoft en colaboración con Microsoft Student Ambassadors.
+¡Esta es una colección de presentaciones que puedes usar, adaptar, y modificar para *tu* propia presentación! Esta construida por Cloud Advocates de Microsoft en colaboración con los Microsoft Learn Student Ambassadors.
 
-🚀 Present in 3 easy steps:
+🚀 Presenta en 3 pasos sencillos:
+1. Encuentra un tema que está listo para usar de [Inteligencia Artificial/Machine Learing](#-ai-and-ml), [Datos](#-data-and-data-science), o [Desarrollo Web](#-web-development). Por ejemplo: [Introducción a Aplicaciones Web Estáticas](./short/intro-static-web-apps/README.md)
```

Comment:
todos estos anchors no funcionan. Por ejemplo `#-ai-and-ml` no funciona porque el titulo cambio a "Inteligencia Artificial y Machine Learning". Hay que probar pero me parece que `#-inteligencia-artificial-y-machine-learning` seria el cambio correcto